frontend/app_flowy/lib/user/presentation/widgets/background.dart

```diff
@@ -49,11 +46,7 @@ class FlowyLogoTitle extends StatelessWidget {
           const VSpace(30),
           Text(
             title,
-            style: TextStyle(
-              color: theme.textColor,
-              fontWeight: FontWeight.w600,
-              fontSize: 24,
-            ),
+            style: Theme.of(context).textTheme.headline5,
```

Comment:
Is "Theme.of(context).textTheme.headline5"

 is equal to 

"TextStyle(
              color: theme.textColor,
              fontWeight: FontWeight.w600,
              fontSize: 24,
            )
"?

Reply:
I think so, https://api.flutter.dev/flutter/material/TextTheme-class.html (font weight is probably `w400`). Seems like `headline6` is more suitable. I think we need a more consistent text styling.

Reply:
So the Theme.of(context).textTheme can handle the font style color without having to hard code it?

Reply:
@rileyhawk1417, sorry I didn't saw your comment. Yep, I mean that is the purpose of theming :smile_cat:, consistency and reduce unnecessary hard coded styles. We could follow Material Design preset (the link) or we could customize it with `textTheme` property inside `ThemeData`.